{{cookiecutter.repo_name}}/CONTRIBUTING.rst

```diff
@@ -91,6 +91,9 @@ Now you can make your changes locally.
     # to run coverage test only, run:
     $ pipenv run coverage
 
+    # to run the pytest tests, run:
+    $ pipenv run pytest
```

Comment:
may be worth saying that `pipenv run test` does the same thing?
Is there any need to keep the shortcut?